src/features/user/UserProfile/components/RegisterTrees.tsx

```diff
@@ -246,203 +246,193 @@ export default function RegisterTrees({}: Props) {
-                      userLocation={userLocation}
-                    />
-                  ) : (
-                    <MapGL
-                      {...mapState}
-                      {...viewport}
-                      onViewportChange={_onViewportChange}
-                      onStateChange={_onStateChange}
-                      onClick={(event) => {
-                        setplantLocation(event.lngLat);
-                        setGeometry({
-                          type: 'Point',
-                          coordinates: event.lngLat,
-                        });
-                        setViewPort({
-                          ...viewport,
-                          latitude: event.lngLat[1],
-                          longitude: event.lngLat[0],
-                          transitionDuration: 400,
-                          transitionInterpolator: new FlyToInterpolator(),
-                          transitionEasing: d3.easeCubic,
-                        });
-                      }}
-                      mapOptions={{
-                        customAttribution:
-                          '<a href="https://www.openstreetmap.org/copyright">© OpenStreetMap contributors</a>',
-                      }}
+                <div className={styles.formFieldHalf}>
+                  <ThemeProvider theme={materialTheme}>
+                    <MuiPickersUtilsProvider
+                      utils={DateFnsUtils}
+                      locale={
+                        localeMapForDate[userLang]
+                          ? localeMapForDate[userLang]
+                          : localeMapForDate['en']
+                      }
                     >
-                      {plantLocation ? (
-                        <Marker
-                          latitude={plantLocation[1]}
-                          longitude={plantLocation[0]}
-                          offsetLeft={5}
-                          offsetTop={-16}
-                          style={{ left: '28px' }}
-                        >
-                          <div className={styles.marker}></div>
-                        </Marker>
-                      ) : null}
-                      <div className={styles.mapNavigation}>
-                        <NavigationControl showCompass={false} />
-                      </div>
-                    </MapGL>
-                  )}
+                      <Controller
+                        render={(properties) => (
+                          <DatePicker
+                            label={t('me:datePlanted')}
+                            value={properties.value}
+                            onChange={properties.onChange}
+                            inputVariant="outlined"
+                            TextFieldComponent={MaterialTextField}
+                            autoOk
+                            disableFuture
+                            minDate={new Date(new Date().setFullYear(1950))}
+                            format="MMMM d, yyyy"
```

Comment:
This is not the preferred date format for Germany. I am not sure if we have already localized date formats yet in any other part of the our web app code (at least we did it for the native app and the old web client)? If yes, we should adapt it the same way, otherwise introducing language depending date formatting can be introduced in another PR e.g. by using i18n.

Reply:
We have a function `formatDate()` that I believe formats the dates. But here in the date picker I think this is where we define the locale, not sure if it is working - https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1116/files/#:~:text=%3CMuiPickersUtilsProvider,%3E

Reply:
Let's discuss this in a new issue as this is not exclusive to this PR